### PR TITLE
[Feature] Add some github actions for issue management

### PR DESCRIPTION
**Pull request by tonylu0728**
add some github actions for issue management

### CODE REVIEW

File: .github/workflows/desktop [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- name: Find issues
```
Comment (yl-1993) [inline .github/workflows/desktop:1]: Why the file name is `desktop`? It looks like `find-issues.yml` is more appropriate.

File: .github/workflows/auto-reply.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      - name: enhancement
+        if: github.event.label.name == 'enhancement'
+        uses: actions-cool/issues-helper@v1.2
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. welcome to commit your issue here , we will check and reply you as soon.
+            你好 @${{ github.event.issue.user.login }}，非常欢迎提交你的问题和需求，我们会尽快确认并回复你~~
```
Comment (yl-1993) [inline .github/workflows/auto-reply.yml:51]: Seems not need to use `if` as the reply content for different label is the same? Meaning that 4 steps (L9-L51) can be compressed into 1 step.

File: .github/workflows/auto-reply.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+      - name: question
+        if: github.event.label.name == 'question'
+        uses: actions-cool/issues-helper@v1.2
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. welcome to commit your issue here , we will check and reply you as soon.
```
Comment (yl-1993) [inline .github/workflows/auto-reply.yml:39]: Hello @${{ github.event.issue.user.login }}. Welcome to commit your issue here, we will check and reply to you as soon as possible.

File: .github/workflows/desktop (hunk 1 above)
Comment (tonylu0728) [reply .github/workflows/desktop]: have modified it, pl check new commit

File: .github/workflows/auto-reply.yml (hunk 2 above)
Comment (tonylu0728) [reply .github/workflows/auto-reply.yml]: have modified it, pl check new commit

**Comment by yl-1993** [comment]
Please remove [.find-comments.yml.swp](https://github.com/openxrlab/xrmocap/pull/28/files#diff-1529f625af438d4cea4b2e8b424bbd2ec2addc68a6ccbf39677517056988bdb7)

File: .github/workflows/close-issues.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: Check need info
```
Comment (yl-1993) [inline .github/workflows/close-issues.yml:1]: Check need info -> Close issues?

File: .github/workflows/find-comments.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+name: check comments
```
Comment (yl-1993) [inline .github/workflows/find-comments.yml:1]: check comments -> Find comments?

File: .github/workflows/find-issues.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+name: check comments
```
Comment (yl-1993) [inline .github/workflows/find-issues.yml:1]: check comments -> Find issues

File: .github/workflows/welcome.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    steps:
+      - name: welcome
+        uses: actions-cool/issues-helper@v2.2.1
+        with:
+          actions: 'welcome'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: hi @${{ github.event.issue.user.login }}, welcome! to commit your issue here , we will check and reply you as soon.
+          labels: 'welcome1, welcome2'
+          assignees: 'xx1'
+          issue-contents: '+1, -1, eyes'
```
Comment (yl-1993) [inline .github/workflows/welcome.yml:19]: It looks like these three lines are used for testing? Please revise them accordingly.

File: .github/workflows/auto-reply.yml [hunk 8]
```diff
@@ -0,0 +1,18 @@
+name: Issue Reply
+on:
+  issues:
+    types: [labeled]
+jobs:
+  reply-helper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: help wanted
+        if: github.event.label.name = 'help wanted' or
```
Comment (yl-1993) [inline .github/workflows/auto-reply.yml:10]: It looks like an unfinished line? Nothing appears after `or`.

**Review by yl-1993 — APPROVED** [review]

**Pull request merged**